docs/sink-configuration.md

```diff
@@ -29,13 +26,11 @@ The following options are available:
 * `db` - InfluxDB Database name (default: `k8s`)
 * `secure` - Connect securely to InfluxDB (default: `false`)
 
-
 ### Google Cloud Monitoring
 This sink supports monitoring metrics only.
 To use the GCM sink add the following flag:
-```
---sink=gcm
-```
+
+	--sink=gcm
```

Comment:
You can use four spaces or a tab to do code blocks just like you can with surroinding text with three backticks.

This is especially useful for very few blocks, since it requires two less lines and it's more readable in plaintext.